# Field that contains the speech specifiers
temoto_2/speechSpecifier[] speechSpecifiers

# Id will be used for making a connection between the
# clients and required services. If id field is left empty
# then Human Context will provide it
string id

---

# Id. Echo of provided id. If there was none provided, then
# Human Context will generate one
string id

# name of the package
string name

# name of the executable/launchable/runnable
string executable

# return the name of the topic where the task should subscribe to
string topic

# return code
#		0: Success
#		1: Failure
#	 -1: Undefined
int64 code

# return message
string message
